lib/internal/crypto/hash.js

```diff
@@ -140,7 +146,74 @@ Hmac.prototype.digest = function digest(outputEncoding) {
 Hmac.prototype._flush = Hash.prototype._flush;
 Hmac.prototype._transform = Hash.prototype._transform;
 
+class Mac extends LazyTransform {
+  constructor(mac, ...args) {
```

Comment:
A code doc indicating what the expected arguments are here would be helpful for others who may need to come in and edit this later

Reply:
I'm going to leave that out. It's clear enough from reading the code how `args` is decomposed. A comment will only get out of sync.